main-cookbooks/teracy-dev/recipes/default.rb

```diff
@@ -38,3 +38,4 @@
 include_recipe 'teracy-dev::docker_machine'
 include_recipe 'teracy-dev::inotify'
 include_recipe 'teracy-dev::proxy'
+include_recipe 'teracy-dev::misc'
```

Comment:
call its with specific function, "teracy-dev::public_mac_address" instead of too general "misc", don't introduce generic name into recipes.